src/utils/utils.ts

```diff
@@ -43,11 +48,26 @@ export function newTracesExploration(
   return new TraceExploration({
     initialDS,
     initialFilters: initialFilters ?? [primarySignalOptions[0].filter],
-    $timeRange: new SceneTimeRange({ from: 'now-15m', to: 'now' }),
+    $timeRange: new SceneTimeRange({ from: 'now-30m', to: 'now' }),
     locationService,
   });
 }
 
+export function newHome(initialDS?: string): Home {
+  return new Home({
+    initialDS,
+    $timeRange: new SceneTimeRange({ from: 'now-30m', to: 'now' }),
+  });
+}
+
+export function getErrorMessage(data: SceneDataState) {
+  return data?.data?.error?.message ?? 'There are no Tempo data sources';
+}
+
+export function getNoDataMessage(context: string) {
```

Comment:
When hovering over the icon it shows a pointer cursors but it's not interactive

![copy](https://github.com/user-attachments/assets/ad6b7741-5a67-4519-9a3b-5cf03ccb39ca)